framework/jaif-inference-scripts/insert-annotations.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+
```

Comment:
Nice, this is great. Does the insert-annotations-to-source currently supports receiving a set of .java and .jaif files as argument?